code-reviews/example1/styles.css

```diff
@@ -0,0 +1,15 @@
+.main {
+    background-color: #add8e6;
+    color: black;
+    padding: 12px;
+}
+
+.list {
+    font-weight: bold;
+    margin-bottom: 4;
+}
+
+recipes {
```

Comment:
Will the styles really apply when the class "styles" is referenced without a dot at the start?